test/src/unit-cppapi-schema-evolution.cc

```diff
@@ -868,17 +870,35 @@ TEST_CASE(
 
   // Read the array
   std::string buffer;
-  std::vector<uint64_t> offsets = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+  std::vector<uint64_t> offsets(10);
   Array array_r(ctx, array_uri, TILEDB_READ);
-  Subarray subarray(ctx, array_r);
-  subarray.add_range(0, 1, 10);
+  Subarray subarray_r(ctx, array_r);
+  subarray_r.add_range(0, 1, 10);
   Query query_r(ctx, array_r, TILEDB_READ);
   query_r.set_layout(layout)
-      .set_subarray(subarray)
+      .set_subarray(subarray_r)
       .set_data_buffer("a", buffer)
       .set_offsets_buffer("a", offsets);
   query_r.submit();
   array_r.close();
```

Comment:
Should we `CHECK` something here?

Reply:
`CHECK(buffer.size() == 0);` seems to work for me.
Should we add it here?